spec/filters/grok_spec.rb

```diff
@@ -701,7 +703,8 @@ def self.sample(message, &block)
       expect( LogStash::Json.dump(event.get('username')) ).to eql "\"testuser\""
 
       expect( event.to_json ).to match %r|"src_ip":"1.1.1.1"|
-      expect( event.to_json ).to match %r|"@timestamp":"20\d\d-\d\d-\d\dT\d\d:\d\d:\d\d\.\d\d\dZ"|
+      # expect( event.to_json ).to match %r|"@timestamp":"20\d\d-\d\d-\d\dT\d\d:\d\d:\d\d(\.(\d\d\d){1,3})?Z"|
```

Comment:
Did you intend to leave this commented out?

Reply:
I intended to delete it. The next line replicates what it was attempting to validate.

Reply:
```suggestion
```